### PR TITLE
Master

### PR DESCRIPTION
**Pull request by River-b**
* **Please check if the PR fulfills these requirements**
- [] The changes have been tested locally
- [] There are no breaking changes

* **What kind of change does this PR introduce?**

* **What is the current behavior?**

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: Documentation/Logo.md [hunk 1]
```diff
@@ -5,28 +5,31 @@ By default, there is _not_ one included in the firmware. This means that once fl
...
 
-- `python img2logo.py infile.png out -m` for Miniware
-- `python img2logo.py infile.png out -p` for Pinecil
+- `python img2logo.py infile.png out -m`   for Miniware
+- `python img2logo.py infile.png out -p`   for Pinecil
 
-Run `python img2logo.py --help` to see available options. Replace word python with python3 if you have multiple versions of python installed.
-Make sure your image file is in the same folder as scripts img2logo.py, output_dfu.py, and output_hex.py.
+Run `python img2logo.py --help` to see available options. Replace the word python with python3 if you have multiple versions of python installed. 
+_Note Bene_: make sure your image file is in the same folder as script files (img2logo.py, output_dfu.py, output_hex.py).
```
Comment (Ralim) [inline Documentation/Logo.md:32]: Isn't this `Nota Bene`.
Though it might be more translatable across languages as just "Note:"

**Review by Ralim — CHANGES_REQUESTED** [review]

File: Documentation/Logo.md (hunk 1 above)
Comment (River-b) [reply Documentation/Logo.md:32]: it is latin like saying Quid Pro Quo which is also latin.   so it's commonly used just like  Quid Pro quo  but I can change it to simple Note word.   Note Bene  means important note.

Comment (River-b) [reply Documentation/Logo.md:32]: oh I see I misspelled it.  you are right I meant to type Nota Bene  that was a typo.  but I just changed it  to be simply Note:

Comment (River-b) [reply Documentation/Logo.md:32]: also I figured out how to change my Default from master to dev  for document updates.

**Pull request closed without merging**